pom.xml

```diff
@@ -29,6 +29,7 @@
         <hamcrest.version>3.0</hamcrest.version>
         <rest-assured.version>5.5.0</rest-assured.version>
         <testcontainers.version>1.20.1</testcontainers.version>
+        <modelmapper.version>3.2.0</modelmapper.version>
```

Comment:
This dependency version should be moved up to jackson and lombok for dependency organisation